<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateTempListWindow</class>
 <widget class="QDialog" name="UpdateTempListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpdateTempListWindow</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>221</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Item 정보 검색하러 가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>물품 Update 정보 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>191</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 사진 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 이름 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 유형 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 용도 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>구매일 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 갯수 : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>homeButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateTempListWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateTempListWindow</receiver>
   <slot>view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateTempListWindow</receiver>
   <slot>updateItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateTempListWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exit()</slot>
  <slot>view()</slot>
  <slot>updateItem()</slot>
  <slot>clear()</slot>
 </slots>
</ui>
